pages/_document.tsx

```diff
@@ -17,6 +18,18 @@ export default function Document() {
       </style>
       <Head>
         <meta name="description" content="Research Catalog | NYPL" />
+        <script async src={appConfig.adobeEmbedUrl[appConfig.environment]} />
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `
+              // First define the global variable for the entire data layer array
+              window.adobeDataLayer = window.adobeDataLayer || [];
+              // Then push in the variables required in the Initial Data Layer Definition
+              window.adobeDataLayer.push({disable_page_view: true});
+              ${appConfig.environment}
```

Comment:
Why is this here?

Reply:
Good catch, had that in there for debugging and forgot to remove